Test.cpp

```diff
@@ -47,6 +47,18 @@
 #define TEST_FUNCTION_LBL "TEST_FUNCTION"
 #endif
 
+#ifndef LLVM_TOOLS_BINARY_DIR
+#define LLVM_TOOLS_BINARY_DIR "/usr/bin"
+#endif
+
+#ifndef CLANG_EXE_NAME
+#define CLANG_EXE_NAME "clang"
+#endif
+
+#ifndef OPT_EXE_FULLPATH
+#define OPT_EXE_FULLPATH "/usr/bin/opt"
+#endif
+
```

Comment:
I think these define (and previous ones above) might be better suited to become `const std::string`s initialised with the define values.

Reply:
I agree on having the checks in cmake. I was mostly arguing in favour of having define-initialised constants rather than simple defines in the code.

Reply:
To be honest, I have add them to make intellisense happy (which otherwise couldn't find the variable definition) :smile: